docs/website/docs/overview/configure.md

```diff
@@ -172,6 +172,19 @@ You can use the `--force` (or `-f`) flag to force the deletion of the registry w
 To update a registry, you can delete it and add it again with the updated value.
 :::
 
+## Ignoring files to synchronize
```

Comment:
Wondering why this is under "Configuration", as, to me, it does not relate to the `odo` configuration, but depends on each project..
Perhaps a dedicated page for this (under "User Guides" or "User Guides > Advanced Usage") would make more sense, no?

Reply:
Or in the page "Only Push Specific Files" in "User Guides > Advanced Usage". WDYT @rm3l ?

Reply:
Ah yes, good point. That would be better, indeed. Thanks.